library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt

```diff
@@ -96,31 +96,27 @@ class SwipeToDismissHandler(
                 .setDuration(ANIMATION_DURATION)
                 .setInterpolator(AccelerateInterpolator())
                 .setUpdateListener { onSwipeViewMove(swipeView.translationY, translationLimit) }
-                .setAnimatorListener(onAnimationEnd = {
+                .setAnimatorEndListener {
                     if (translationTo != 0f) {
                         onDismiss()
                     }
 
                     // remove the update listener, otherwise it will be saved on the next animation execution:
                     swipeView.animate().setUpdateListener(null)
-                })
+                }
                 .start()
     }
 }
 
-internal fun ViewPropertyAnimator.setAnimatorListener(
-        onAnimationEnd: ((Animator?) -> Unit)? = null,
-        onAnimationStart: ((Animator?) -> Unit)? = null
-) = this.setListener(
+private fun ViewPropertyAnimator.setAnimatorEndListener(
+        onAnimationEnd: () -> Unit,
```

Comment:
Not a blocker for the PR but don't you think you should pass the `Animator` inside the lambda, even if it is not used? (Just to keep the API behaviour)

Reply:
it's just a private extension, so let's simplify it.